<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InicioSesion</class>
 <widget class="QWidget" name="InicioSesion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/recursos/Img/pagina-de-inicio-de-sesion.png</normaloff>:/recursos/Img/pagina-de-inicio-de-sesion.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="botInicioSesion">
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Haettenschweiler&quot;;
letter-spacing: 1.2px;</string>
     </property>
     <property name="text">
      <string>Iniciar sesion</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Montserrat&quot;;
color: white;
font-style: italic;
</string>
       </property>
       <property name="text">
        <string>Usuario</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Montserrat&quot;;
color: white;
font-style: italic;</string>
       </property>
       <property name="text">
        <string>Contrase√±a</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="contrasenaLineEdit">
       <property name="styleSheet">
        <string notr="true">height: 35px;
font-size: 14px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="usuarioLineEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">height: 35px;
font-size: 14px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
